packages/ndla-ui/src/SearchTypeResult/SearchItem.tsx

```diff
@@ -62,41 +62,26 @@ const ItemWrapper = styled.div`
   border: 1px solid ${colors.brand.neutral7};
   border-radius: 5px;
   transition: all ${animations.durations.fast} ease-in-out;
-  &:hover {
-    height: calc(100% + 4px);
-    width: calc(100% + 4px);
-    .topic-header-image {
-      height: calc(100% + 6px);
-      width: calc(100% + 6px);
+  .resource-no-image {
+    background-color: ${colors.brand.greyLightest};
+    .c-content-type-badge {
+      width: 26px;
+      height: 26px;
+      left: 45px;
+      margin-left: 0;
+      top: 100%;
+      margin-top: -13px;
+      opacity: 1;
     }
-    .topic-label {
-      svg {
-        transform: scale(1.2);
-      }
-    }
-    .topic-label-list {
-      margin-left: 2px;
-    }
-    .resource-type-wrapper {
-      padding: 0 calc(${spacing.normal} + 2px);
-      .resource-icon-wrapper {
-        left: 19px;
-        svg {
-          transform: scale(1.2);
-        }
-      }
-    }
-    .resource-no-image {
-      background-color: ${colors.brand.greyLightest};
-      .c-content-type-badge {
-        width: 26px;
-        height: 26px;
-        left: 45px;
-        margin-left: 0;
-        top: 100%;
-        margin-top: -13px;
-        opacity: 1;
-      }
+  }
+
+  :hover {
+    img {
+      transform: scale(1.1);
+      opacity: 1.2;
+
+      transition: all ${animations.durations.fast} ease-in-out;
```

Comment:
transition må alltid eksistere for at a transition out skal fungere. Nå forsvinner den umiddelbart når hover er over, og dermed "blinker" bildet tilbake til sin opprinnelige tilstand